fastlane/lib/fastlane/fast_file.rb

```diff
@@ -1,3 +1,26 @@
+require "rubygems"
+
+class String
+  # Gem::Version.new barfs on things like "v0.1.0", which is the style
+  # generated by the rake release task. Just strip off any initial v
+  # to generate a Gem::Version from a tag.
+  def version_number_from_tag
```

Comment:
Let's prefix the method with `fastlane_` to avoid name conflicts with other libraries.